src/detection/card/action/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Button } from '@material-ui/core'
+
+import { Actions } from 'detection/card/action/actions'
+
+export const Buttons = () => <Actions>
+  <Button variant={'contained'} color={'primary'}>Set status</Button>
```

Comment:
Since these buttons act on vastly different actions, we should maybe give them different colors? For example, making the `Delete` button red seems sensible.